src/Stripe.net/Services/Accounts/AccountCapabilitiesAuBecsDebitPaymentsOptions.cs

```diff
@@ -4,6 +4,11 @@ namespace Stripe
 
     public class AccountCapabilitiesAuBecsDebitPaymentsOptions : INestedOptions
     {
+        /// <summary>
+        /// Passing true requests the capability for the account, if it is not already requested. A
```

Comment:
I am guessing we can't detect this should be `<c>true</c>` right? Or we need to fix the docstring to be have the ` around true internally?

Reply:
Correct.